circle.yml

```diff
@@ -627,7 +627,7 @@ jobs:
             JOBS: 1 # https://github.com/mapbox/mapbox-gl-native/issues/9108
           command: |
             xvfb-run --server-args="-screen 0 1024x768x24" \
-              scripts/valgrind.sh build/qt-linux-x86_64/Release/mbgl-test --gtest_filter=-*.Load --gtest_filter=-Memory.Vector
+              build/qt-linux-x86_64/Release/mbgl-test --gtest_filter=-*.Load --gtest_filter=-Memory.Vector
```

Comment:
Had to remote valgrind since it's unable to run with `llvmpipe`'s generated code: https://circleci.com/gh/mapbox/mapbox-gl-native/40799

Using `softpipe` instead of `llvmpipe` resulted in a deadlock.